src/client/worker.rs

```diff
@@ -1,72 +1,64 @@
-use cli_table::format::Justify;
-use cli_table::{print_stdout, Cell, CellStruct, Color, Style, Table};
+use serde::{Deserialize, Serialize};
 
-use crate::client::globalsettings::GlobalSettings;
 use crate::common::manager::info::GetManagerInfo;
 use crate::transfer::messages::{LostWorkerReasonInfo, WorkerExitInfo, WorkerInfo};
+use crate::WorkerId;
+
+use std::string::String;
 
 pub enum WorkerState {
     Running,
     ConnectionLost,
     HeartbeatLost,
 }
 
-fn worker_state(worker: &WorkerInfo) -> CellStruct {
-    match worker.ended {
-        None => "RUNNING".cell().foreground_color(Some(Color::Green)),
-        Some(WorkerExitInfo {
-            reason: LostWorkerReasonInfo::ConnectionLost,
-            ..
-        }) => "CONNECTION LOST".cell().foreground_color(Some(Color::Red)),
-        Some(WorkerExitInfo {
-            reason: LostWorkerReasonInfo::HeartbeatLost,
-            ..
-        }) => "HEARTBEAT LOST".cell().foreground_color(Some(Color::Red)),
-        Some(WorkerExitInfo {
-            reason: LostWorkerReasonInfo::IdleTimeout,
-            ..
-        }) => "IDLE TIMEOUT".cell().foreground_color(Some(Color::Cyan)),
-        Some(WorkerExitInfo {
-            reason: LostWorkerReasonInfo::Stopped,
-            ..
-        }) => "STOPPED".cell().foreground_color(Some(Color::Magenta)),
-    }
+#[derive(Serialize, Deserialize)]
```

Comment:
What is the purpose of this struct?

Reply:
This is a leftover that is currently not needed, since it was used for Json outputting by serde_json.